src/mbgl/map/map_impl.cpp

```diff
@@ -154,7 +154,7 @@ void Map::Impl::onWillStartRenderingMap() {
 }
 
 void Map::Impl::onDidFinishRenderingMap() {
-    if (mode == MapMode::Continuous && loading) {
```

Comment:
why is this change required?

Reply:
I removed `loading` because it is checked again two lines below. Having the check here prevents sending `onDidFinishRenderingMap` event again after loading new tiles caused by panning or zooming the map.